_data/nav.yml

```diff
@@ -0,0 +1,26 @@
+default:
+  footer:
+    agencies:
+      become_a_partner: 'base_url/partners/'
```

Comment:
what if we redid this as `BASE_URL` or `$BASE_URL` or something to make it more obvious it's going to be replaced later on in the process?

Reply:
> what if we redid this as `BASE_URL` or `$BASE_URL` or something to make it more obvious it's going to be replaced later on in the process?

Yeah that's a good idea.

Reply:
> what if we redid this as `BASE_URL` or `$BASE_URL` or something to make it more obvious it's going to be replaced later on in the process?

Updated in 53a272df